src/action/ActionMap.js

```diff
@@ -0,0 +1,126 @@
+/**
+ *  ActionMap maps input semantic paths to {@link Action}s and/or {@link Filter}s.
+ *
+ */
+export default class ActionMap {
+  /**
+   * @param {Array[]} filters An array of [semanticPath, Filter]
+   * @parm {string} url An optional URL to JSON data holding binding info
+   */
+  constructor(filters, url = null) {
```

Comment:
I'd expect a way to provide the configuration without need to load an URL, like sending an javascript object/json programatically

Reply:
My thought is that by far most common case will be loading by URL so it's an optional parameter to the constructor. If the user has a data structure they could do:
    let actionMap = new ActionMap(filters)
    actionMap.loadData(dataStructure)
If we make the constructor have a data parameter instead of a url it seems like we're making the common case harder and the rare case easier.